shuffle/cli/src/account.rs

```diff
@@ -13,17 +13,42 @@ use diem_sdk::{
 use diem_types::{
     account_config, chain_id::ChainId, transaction::authenticator::AuthenticationKey,
 };
+use dir::home_dir;
 use generate_key::load_key;
 use shuffle_transaction_builder::framework::encode_create_parent_vasp_account_script_function;
-use std::path::PathBuf;
+use std::path::{Path};
 
-pub fn handle(project_dir: PathBuf, account_key_path: PathBuf) -> Result<()> {
-    let config_path = project_dir.join("nodeconfig/0").join("node.yaml");
+
+pub fn handle(rotate : bool) -> Result<()> {
```

Comment:
handle what? would be great to have a comment above here describing a little bit of what this is handling

Reply:
yes. Just fyi, that handles the new account creation based off of a randomly generated key. Will add a comment that reflects this.

Reply:
I set this precedent in `main.rs` and it reads better when prefixed by the module (`account::handle`). In this case the `handle` is referring to the module it's in: `account.rs`. I'm open to other ideas.

https://github.com/diem/diem/blob/9a04ed9fa8695f1e6b1175af97d4d10e0fc6ed1b/shuffle/cli/src/main.rs#L16-L26